lib/xnvme_be_cbi_admin_shim.c

```diff
@@ -11,7 +11,6 @@
 #include <errno.h>
 #include <sys/stat.h>
 #include <unistd.h>
-#include <libxnvme_spec_fs.h>
```

Comment:
This needs a libxnvme.h

Reply:
Since the internal headers still include other headers, then it gets ``libxnvme.h`` via ``xnvme_be.h``